irods/models.py

```diff
@@ -1,14 +1,23 @@
+from __future__ import absolute_import
```

Comment:
what is this?

Reply:
It is unnecessary in python2.7+ or Python3, rather it's a directive to make Python2.5 interpreters act more like Python2.7+ as regards [prohibiting implicit relative imports](https://stackoverflow.com/questions/22678919/what-is-absolute-import-in-python)